Sources/Plasma/FeatureLib/pfMetalPipeline/ShaderSrc/FixedPipelineShaders.metal

```diff
@@ -445,7 +472,7 @@ fragment half4 pipelineFragmentShader(ColorInOut in [[stage_in]],
             }
         }
         
-        currentColor = half4(in.vtxColor.rgb, 1.0h) * currentColor;
```

Comment:
Might be a good time to ask about this one. After blending the layers - the shader again multiplies by the vertex +lighting color. This was carried over from the original GL shaders. I know if this isn't here the output is wrong. But I'm not sure why.

Reply:
Skimming through the code, my guess is this:
`currentColor` starts out set to the lighting colour with no material values, but then on the first layer is overwritten to the layer colours of the first layer (in `blendFirst`) with subsequent layers blending into that. Since we've lost the lighting colours at that point, we need to multiply them back in.